File: backend/src/main/java/com/backend/domain/repository/dto/response/github/ReadmeResponse.java
package com.backend.domain.repository.dto.response.github;

import com.fasterxml.jackson.annotation.JsonProperty;

// readme 응답용 DTO
public record ReadmeResponse(
        String name,
        String path,
        String sha,
        Integer size,
        String url,
        @JsonProperty("html_url") String htmlUrl,
        @JsonProperty("download_url") String downloadUrl,
        String type,
        String content,
        String encoding
) {
}

File: backend/src/main/java/com/backend/domain/repository/dto/response/github/ReadmeResponse.kt
package com.backend.domain.repository.dto.response.github

// readme 응답용 DTO
data class ReadmeResponse(
    val name: String?,
    val path: String?,
    val sha: String?,
    val size: Int?,
    val url: String?,
    val htmlUrl: String?,
    val downloadUrl: String?,
    val type: String?,
    val content: String?,
    val encoding: String?
)
